message-index/messages/GHC-51179/index.md

```diff
@@ -0,0 +1,19 @@
+---
+ title: Missing LambdaCase
+ summary:  Missing LambdaCase language extension
+ severity: error
+ introduced: 9.6.1
+ extension: LambdaCase
+ ---
+
+[Lambda Case extension](https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/lambda_case.html)
```

Comment:
```suggestion
The [Lambda Case extension](https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/lambda_case.html)
```

Reply:
ok

Reply:
rephrased to:
"The [Lambda Case extension](https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/lambda_case.html)
 enables a syntactic sugar for a lambda function that pattern matches its argument immediately."